package/src/tools_pex.rs

```diff
@@ -58,7 +58,7 @@ pub(crate) fn build_tools_pex(
                     "--style",
                     "universal",
                     "--pip-version",
-                    "22.3",
+                    "24.3.1",
```

Comment:
isnt pip on 25.x?

Reply:
Yeah, not trying for the latest and greatest, just something less archaic.